setup.cfg

```
@@ -36,6 +36,7 @@ dev =
     bandit==1.6.2
     black==19.10b0
     djehouty==0.1.5
+    faker==4.0.1
```

It should probably work, but, as referred in PyPI, this package's name starts with an uppercase F:

```suggestion
    Faker==4.0.1
```

---

Thanks, good catch